collect_app/src/androidTest/java/org/odk/collect/android/support/CollectHelpers.kt

```diff
@@ -16,4 +20,21 @@ object CollectHelpers {
         application.component = testComponent
         return testComponent
     }
+
+    @JvmStatic
+    fun addGDProject(gdProject: Project.Saved, accountName: String, testDependencies: TestDependencies) {
```

Comment:
This should probably take a `Project.New` so we don't end up with any problems caused by specifying IDs and not letting the repo generate them itself.

Reply:
Right, fixed.